examples/js_dom_cube/README.md

```diff
@@ -1,150 +1,148 @@
-        '.wav': 'audio/wav',
-        '.mp4': 'video/mp4',
-        '.woff': 'application/font-woff',
-        '.ttf': 'application/font-ttf',
-        '.eot': 'application/vnd.ms-fontobject',
-        '.otf': 'application/font-otf',
-        '.wasm': 'application/wasm'
-    };
-
-    var contentType = mimeTypes[extname] || 'application/octet-stream';
-
-    fs.readFile(filePath, function(error, content) {
-        if (error) {
-            if(error.code == 'ENOENT') {
-                fs.readFile('./404.html', function(error, content) {
-                    res.writeHead(404, { 'Content-Type': 'text/html' });
-                    res.end(content, 'utf-8');
-                });
-            }
-            else {
-                res.writeHead(500);
-                res.end('Sorry, check with the site admin for error: '+error.code+' ..\n');
-            }
-        }
-        else {
-            res.writeHead(200, { 'Content-Type': contentType });
-            res.end(content, 'utf-8');
-        }
-    });
+	console.log('[route] - ', req.url);
+
+	var filePath = '.' + req.url;
+	if (filePath == './') {
+		filePath = './index.html';
+	}
+
+	var extname = String(path.extname(filePath)).toLowerCase();
+	var mimeTypes = {
+		'.html': 'text/html',
+		'.js': 'text/javascript',
+		'.css': 'text/css',
+		'.json': 'application/json',
+		'.png': 'image/png',
+		'.jpg': 'image/jpg',
+		'.gif': 'image/gif',
+		'.svg': 'image/svg+xml',
+		'.wav': 'audio/wav',
+		'.mp4': 'video/mp4',
+		'.woff': 'application/font-woff',
+		'.ttf': 'application/font-ttf',
+		'.eot': 'application/vnd.ms-fontobject',
+		'.otf': 'application/font-otf',
+		'.wasm': 'application/wasm',
+	};
+
+	var contentType = mimeTypes[extname] || 'application/octet-stream';
+
+	fs.readFile(filePath, function (error, content) {
+		if (error) {
+			if (error.code == 'ENOENT') {
+				fs.readFile('./404.html', function (error, content) {
+					res.writeHead(404, { 'Content-Type': 'text/html' });
+					res.end(content, 'utf-8');
+				});
+			} else {
+				res.writeHead(500);
+				res.end('Sorry, check with the site admin for error: ' + error.code + ' ..\n');
+			}
+		} else {
+			res.writeHead(200, { 'Content-Type': contentType });
+			res.end(content, 'utf-8');
+		}
+	});
 };
 
 const server = http.createServer(reqListener);
 server.listen(port, host, () => {
-  console.log(`Server is running on http://${host}:${port}`);
+	console.log(`Server is running on http://${host}:${port}`);
 });
-
 ```
 
-This project has a no dependence serve in `./server.js` path.
-That can be run with `node` command after build.
-
-or just run: `npm run start`
+Now you can build and run the project with the added scripts.
 
-To install node, you can access node [download page](https://nodejs.org/en/download/) 
-or [package manager](https://nodejs.org/en/download/package-manager)
-
-```
-$ cd examples/js_dom_draw/
-$ npm run build
+```sh
+npm run build
+npm run start
 ```
 
-
 ### V server
-```v ignore
```

Comment:
I removed the `ignore` here and in the draw example. I'm assuming it's covered by the doc CI that is run on the PR and won't interfere when other CI comes in. Please enlighten me if it might cause problems.

The rest of the changes should just be fixes in wording and formatting.

Reply:
I think it would not cause problems. I am not sure what the original reason for ignoring was, the program seems to be a complete valid vweb app 🤷🏻‍♂️ .